File: client/src/libs/graphql/mutations.ts
export const CreateOwnFileFileRecord = `
  mutation CreateOwnFileFileRecord(
    $input: CreateFileInput!
  ) {
    createOwnFile(input: $input) {
      created_at
      file_name
      file_path
      user_id
      type
    }
  }
`

export const DeleteOwnFileRecord = `
  mutation DeleteOwnFileRecord(
    $file_path: String!
  ) {
    deleteOwnFile(file_path: $file_path) {
      file_path
    }
  }
`

File: deepdream-generator-api/deepdream_generator_api/libs/graphql/mutations.py
from gql import gql

create_job = gql(
    """
    mutation CreateJob($input: CreateJobInput!) {
      createJob(input: $input) {
        id
        user_id
        params
        input_path
        input_name
        progress
        result_path
        started_at
      }
    }
"""
)

update_job = gql(
    """
    mutation UpdateJob($input: UpdateJobInput!) {
      updateJob(input: $input) {
        id
        user_id
        params
        input_path
        input_name
        progress
        result_path
        started_at
      }
    }
"""
)

create_file_record = gql(
    """
    mutation CreateFileFileRecord($input: CreateFileInput!) {
      createFile(input: $input) {
        user_id
        file_name
        file_path
        created_at
        type
      }
    }
"""
)
